test/cache-connectorSpec.js

```diff
@@ -261,14 +284,18 @@ describe( 'advanced sets', () => {
   })
 
   it( 'writes multiple values in quick succession to an existing table', ( done ) => {
-    dbConnector.set( 'some-table/itemA', { val: 1 }, () => {})
-    dbConnector.set( 'some-table/itemA', { val: 2 }, () => {})
-    dbConnector.set( 'some-table/itemA', { val: 3 }, () => {})
-    dbConnector.set( 'some-table/itemA', { val: 4 }, () => {})
-    dbConnector.set( 'some-table/itemA', { val: 5 }, ( error ) => {
-      expect( error ).to.be.null
-      done()
-    })
+    let dbConnSet = set.bind(null, dbConnector);
```

Comment:
the previous test was meant to actually see how they all work when called in sync, using promises breaks the test functionality

Reply:
Right... the thing is that there is a race condition I think between testcases...